### PR TITLE
PE-860: Implement our own getData method with retries

### PR DESCRIPTION
**Pull request by fedellen**

### CODE REVIEW

**Review by arielmelendez — APPROVED** [review]

File: src/arfs/arfsdao.ts [hunk 1]
```diff
@@ -1270,9 +1271,26 @@ export class ArFSDAO extends ArFSDAOAnonymous {
...
+							return exponentialDelay(retryNumber);
+						}
+					});
+					const {
+						data: driveDataBuffer
+					}: {
+						data: Buffer;
+					} = await axiosInstance.get(reqURL, {
+						responseType: 'arraybuffer'
+					});
```
Comment (matibat) [inline src/arfs/arfsdao.ts:1293]: This code is pretty similar to the `getDatForTxID(txId)` method. Does it worth consolidating'em into a method `arweaveGETRequestWithRetries(urlPath)` used by both?

**Review by matibat — APPROVED** [review]
Left one single comment, but looks good overall

File: src/arfs/arfsdao.ts (hunk 1 above)
Comment (arielmelendez) [reply src/arfs/arfsdao.ts:1293]: Probably yes :) If I have time for it today, I'll give it a whirl. Otherwise would like to get this into QA.

Comment (fedellen) [reply src/arfs/arfsdao.ts:1293]: Ah, this is in `arfsdao`, was looking for this in the wrong place. Yeah this does seem repetitious here.

**Pull request merged**